web/react/components/textbox.jsx

```diff
@@ -129,12 +129,12 @@ export default class Textbox extends React.Component {
     }
 
     render() {
+        const hastext = this.props.messageText.length > 0;
```

Comment:
Very minor, but could you use camel case for this variable name(aka `hasText` over `hastext`)? It's mentioned in the [style guide here](http://docs.mattermost.com/developer/style-guide.html#variables)